pkgs/servers/monitoring/prometheus/rtl_433-exporter.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub, pkgs }:
+
+buildGoModule rec {
+  name = "rtl_433-exporter";
+  version = "ac2004036f0f8ddd281eaab5dfc955120cc1adf6";
+
+  src = fetchFromGitHub {
+    owner = "mhansen";
+    repo = "rtl_433_prometheus";
+    rev = version;
+    sha256 = "0yz0jv5h9iwc2s136mph8wf056w4zr9vscpzcj21pnd1q6f7wpwf";
+  };
+
+  postPatch = "substituteInPlace rtl_433_prometheus.go --replace /bin/bash ${pkgs.bash}/bin/bash";
+
+  vendorSha256 = "03mnmzq72844hzyw7iq5g4gm1ihpqkg4i9dgj2yln1ghwk843hq6";
+
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Prometheus time-series DB exporter for rtl_433 433MHz radio packet decoder";
+    homepage = "https://github.com/mhansen/rtl_433_prometheus";
+    license = licenses.mit;
```

Comment:
maintainers? If you're not already in the list, please add yourself (in a separate commit).